src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -300,6 +306,19 @@ private static Event GetEvent(LogEvent e)
 			}
 
 			return evnt;
+
+			double? ToDouble()
+			{
+				try
+				{
+					return Convert.ToDouble(elapsedMsObj.Value);
+				}
+				catch
+				{
+					// ignored
```

Comment:
Q: Would it make sense to put a warning here? I feel like returning `null` could have unexpected behaviour for the user? Asking this without knowing all the context.

Reply:
Normally yeah, but since this is after we exhausted all the regular casts its best effort from here on out.